exercises/F-strings-methods/exercise.js

```diff
@@ -1,3 +1,7 @@
 // Start by creating a variable `message`
-
-console.log(message);
+var myName = "My name is ";
+var name = "Ryno";
+var length = " and my name is "
+var nameLength = name.length;
+var characters = " characters long"
+console.log(myName + name + length + nameLength +characters);
```

Comment:
Hey Ryno, I'm sure if you did this over now you would be able to log it out in one line with string interpolation, but nice job man.